bin/i18n/sync-hourofcode-out

```diff
@@ -70,6 +70,9 @@ def restore_sanitized_headers
   Dir.glob("pegasus/sites.v3/hourofcode.com/i18n/public/**/*.md.partial").each do |path|
     source_path = path.sub(/\/i18n\/public\/..\//, "/public/")
     unless File.exist? source_path
+      # Because we give _all_ files coming from crowdin the partial extension,
+      # we can't know for sure whether or not the source did unless we check
```

Comment:
@hamms Can you elaborate more on what you are referring to when you mention "we can't know for sure whether or not the source did ...?  Other than this question, change looks good.